<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SigTCPExport</class>
 <widget class="QDialog" name="SigTCPExport">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>187</width>
    <height>96</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="1">
      <widget class="QSpinBox" name="m_spinBox_port">
       <property name="maximum">
        <number>65535</number>
       </property>
       <property name="value">
        <number>8712</number>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="m_combobox_server_ip"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>端口号</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>IP地址</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="m_pushButton_tcp_export_enable">
       <property name="text">
        <string>Closed</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
